pathfinding_service.rst

```diff
@@ -161,13 +177,16 @@ Example
         "error": "Invalid signature."
     }
 
-``api/1/paths``
-^^^^^^^^^^^^^^^
+``api/1/get_paths``
+^^^^^^^^^^^^^^^^^^^
 
-Request a list of possible paths from startpoint to endpoint for a given transfer value.
+The method will do ``num_path`` iterations of Dijkstra on the last-known state of the Raiden Network (regarded as directed weighted graph) to return num_path different path's for a mediated transfer of _value_.
 
-This method will be rate-limited in a configurable way. If the rate limit is exceeded, clients can be required to pay the path-finding service with RDN tokens via the Raiden Network. The required path for this payment will be provided by the service for free. This enables a simple user experience for light users without the need for additional on-chain transactions for channel creations or payments, while at the same time monetizing extensive use of the API.
```

Comment:
What happened to the comment about rate-limiting and payments?

Reply:
We don't have implemented anything like this yet, but I can put it back in the spec

Reply:
Our idea was to add payment functionality after the initial prototype is built.

Reply:
@err508 It's not implemented yet but we definitely need this rate limiting as pay-per-request imposes an unnecessary restriction for casual users. Paying with RDN would require a RDN balance and a RDN connection (be it µRaiden oder Raiden) to the pathfinding service. We don't want that for "casual" users, only for premium/high throughput users.

Reply:
Added this as a section `future work`.